org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/servlet/HttpProxy.java

```diff
@@ -670,4 +722,46 @@ public void cancelled() {
       }
     };
   }
+
+  /**
+   * Cookie store which uses {@link #m_defaultCookieStore} with all operations run for the specified {@link ISession}.
+   * May be used if async threads access the cookie store where {@link ISession#CURRENT} is not set correctly.
+   */
+  public class SpecificSessionCookieStore implements CookieStore {
```

Comment:
Ist the public modifier useful? The class is not static, thus it cannot be created to create a new instance without a surrounding instance. It might be useful to check "instanceof", but is this really a valid use case? The class also seems not to be very extensible, because some members are private, so we might as well make the whole class private.

Reply:
Thanks, changed to private